### PR TITLE
datadog metrics

### PR DESCRIPTION
**Pull request by benrudolph**
#dsa-test-crush-day

first go at travis metrics in datadog

### CODE REVIEW

File: docker/run.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #! /bin/bash
 # This script runs inside the web container
 set -e
+declare -A metrics
```
Comment (benrudolph) [inline docker/run.sh]: going to cleanup this variable (not needed) in another commit later

**Comment by benrudolph** [comment]
(also happy for people to expand on this during the boston test day, didn't get as far as i had wanted to)

**Comment by benrudolph** [comment]
@snopoke having some difficulty getting this working, and i'm starting to this this is the reason: https://github.com/travis-ci/travis-ci/issues/1500#issuecomment-26122631. what did you end up doing to test your secure vars?

**Comment by snopoke** [comment]
@benrudolph could it be that you also need to export the env var here: https://github.com/dimagi/commcare-hq/blob/c60a6e4405652bc01cba6abd5c5bacc6f0d40ee8/scripts/docker#L176-L176

**Comment by benrudolph** [comment]
(offlined, we're going to try to merge this to see if the variables only get loaded when merged)

**Comment by benrudolph** [comment]
(approved by simon, merging)

**Pull request merged**